packages/camera/camera/example/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ publish_to: none
 
 environment:
   sdk: ">=2.14.0 <3.0.0"
-  flutter: ">=2.10.0"
+  flutter: ">=3.0.0"
```

Comment:
You'll need to change the requirements for `camera` itself too, since we can't guarantee we won't break 2.10 going forward if we can't run the example.

Reply:
Bumped it to Flutter `3.0` also for `camera` 👍  Is it then still just a patch update? For v0.10.0+2 this was case

Reply:
Yes; dropping support for old versions isn't a breaking change because the `pub` resolver just won't even include it.